VocaDbWeb.Core/Controllers/Api/SampleObjects.cs

```diff
@@ -0,0 +1,178 @@
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using VocaDb.Model.DataContracts;
+using VocaDb.Model.DataContracts.Albums;
+using VocaDb.Model.DataContracts.Artists;
+using VocaDb.Model.DataContracts.PVs;
+using VocaDb.Model.DataContracts.Songs;
+using VocaDb.Model.DataContracts.Tags;
+using VocaDb.Model.Domain;
+using VocaDb.Model.Domain.Albums;
+using VocaDb.Model.Domain.Artists;
+using VocaDb.Model.Domain.ExtLinks;
+using VocaDb.Model.Domain.Globalization;
+using VocaDb.Model.Domain.PVs;
+using VocaDb.Model.Domain.Songs;
+using VocaDb.Model.Service;
+
+namespace VocaDb.Web.Areas.HelpPage.App_Start
+{
+	public static class SampleObjects

```

Comment:
What is the purpose of this class?

Reply:
In the API Explorer that we were using before Swagger, there was possibility to specify custom sample data for API requests. Don't know if Swagger supports that. Actually I remembered that we were already using this for Swagger, but guess not.